py3status/module.py

```diff
@@ -220,10 +230,19 @@ def run(self):
                     err = sys.exc_info()[1]
                     syslog(LOG_WARNING,
                            'user method {} failed ({})'.format(meth, err))
-
-            # don't be hasty mate, let's take it easy for now
-            sleep(self.config['interval'])
-
+            # don't be hasty mate
+            # set timer to do update next time one is needed
+            if cache_time:
+                delay = cache_time - time()
```

Comment:
Just to be picky, wouldn't a `delay = max(cache_time - time(), self.config['minimum_interval'])` be better and avoid the condition below ? ;)